docs/CHANGELOG.asciidoc

```diff
@@ -38,15 +38,484 @@ Review the following information about the {kib} 8.3.0 release.
+* Hide job messages clear notifications tooltip on click {kibana-pull}132982[#132982]
+* Filtering category runtime fields in advanced wizard {kibana-pull}132916[#132916]
+* Fixes trained model testing so it is available for users with ML read permissions {kibana-pull}132698[#132698]
+* Wizard validation improvements {kibana-pull}132615[#132615]
+* Adding type for job summary state {kibana-pull}131643[#131643]
+
+Management::
+* Fixes linebreaks (\r\n) mis-applied from history {kibana-pull}131037[#131037]
+* Fixes Kibana DevTool Copy as CURL does not url encode special chars in indice date math {kibana-pull}130970[#130970]
+* Fixes cat APIs returning as escaped string {kibana-pull}130638[#130638]
+* Fixes Elasticsearch doc VIEW IN CONSOLE will clean local Kibana console form history {kibana-pull}127430[#127430]
+
+Maps::
+* Fixes icon markers fail to load when browser zoomed out {kibana-pull}134367[#134367]
+* Hide create filter UI in canvas {kibana-pull}133943[#133943]
+* Use label features from ES vector tile search API to fix multiple labels {kibana-pull}132080[#132080]
+* Fixes Map panels should not show the user controls in a dashboard report {kibana-pull}131970[#131970]
+* Show "no results" found for vector tile aggregations when there are no results {kibana-pull}130821[#130821]
+
+Monitoring::
+* Prevent exceptions in rule when no data present {kibana-pull}131332[#131332]
+* Fixes displaying ES version for external collection {kibana-pull}131194[#131194]
+* Fixes node type detection for external collection {kibana-pull}131156[#131156]
+* Use server.publicBaseUrl in Alert links {kibana-pull}131154[#131154]
+
+Observability::
+* Fixes x-axis on error charts {kibana-pull}134193[#134193]
+* Display ENVIRONMENT_ALL label instead of value {kibana-pull}133616[#133616]
+* Fixes normalizers to not parse list values if they are already parsed {kibana-pull}133563[#133563]
+* Change bucket_scripts to use params for thresholds {kibana-pull}133214[#133214]
+* Use Observability rule type registry for list of rule types {kibana-pull}132484[#132484]
+* APM anomaly rule type should appear in o11y rules page {kibana-pull}132476[#132476]
+* Fixes monitors details page errors {kibana-pull}132196[#132196]
+* Set a valid service_name for python APM onboarding {kibana-pull}131959[#131959]
+* Rename service groups template titles and links {kibana-pull}131381[#131381]
+* Show service group icon only for service groups {kibana-pull}131138[#131138]
+* Refactor Metric Threshold rule to push evaluations to Elasticsearch {kibana-pull}126214[#126214]
+* Ellipsis truncation issue - dependencies and service section {kibana-pull}122203[#122203]
+* Fixes lookback window for anomalies for anomaly alert {kibana-pull}93389[#93389]
+
+Operations::
+Fixes error handling on precommit hook {kibana-pull}132998[#132998]
+
+Platform::
+Retry INIT step when hitting incompatible_cluster_routing_allocation {kibana-pull}131809[#131809]
```

Comment:
It might be more user-friendly to rephrase this in a less technical way:
```suggestion
Prevents Kibana from bootlooping during migrations when Elasticsearch routing allocation settings are incompatible {kibana-pull}131809[#131809]
```